app/models/obituary.rb

```diff
@@ -3,8 +3,25 @@ class Obituary < ApplicationRecord
   belongs_to :user
   before_save :default_image
 
+  def name
+    first_name + ' ' + last_name
+  end
+
+  def location
+    city + ' ' + state
+  end
+
+  def date
+    created_at.strftime("%B %-d, %Y")
+  end
```

Comment:
formatting the data from the database for the view. It is all pretty straightforward. Thought it was easier than changing the database, for instance to have name rather than first and last.

Reply:
Nice work around, I agree I think this is best. plus who doesnt love a fat model?

Reply:
Yeah this looks great Stella, It definitely seems like the right place for all this logic.